src/main/resources/db/migration/V4_4__grant_to_cloudsqliamuser.sql

```diff
@@ -0,0 +1,9 @@
+DO $$
+BEGIN
+  CREATE ROLE cloudsqliamuser WITH NOLOGIN;
```

Comment:
Denne finnes allerede på gcp, men vi må ha den med for at testene ikke skal feile.